README.md

```diff
@@ -11,14 +11,14 @@ Maybe we can transform this into something useful, a more advanced example, just
 
 ## Development
 
-We use `node` version `10.15.0`
+We use `node` version ~`10.15.0`~ `14.9.0`
```

Comment:
```suggestion
We use `node` version `14.9.0`
```
Maybe instead of crossing out the `10.15.0` we just remove it entirely, to remove confusion?